subprojects/jacoco/src/integTest/groovy/org/gradle/testing/jacoco/plugins/fixtures/JavaProjectUnderTest.groovy

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.testing.jacoco.plugins.fixtures
+
+import org.gradle.test.fixtures.file.TestFile
+
+class JavaProjectUnderTest {
```

Comment:
This is a really cool idea. We should do this elsewhere (not a part of this PR of course).